interfaces/group-dao-I.ts

```diff
@@ -1,12 +1,14 @@
 import Group from "../models/messages/group";
+import User from "../models/users/user";
 
 /**
  * @file Declares API for Group related data access object methods
  */
 export default interface GroupDaoI {
     findAllUserGroups (uid: string): Promise<Group[]>;
-    createGroup (creatorUid: string, uids: string[], group: Group): Promise<Group>;
+    createGroup (creatorUid: string, users: User[], group: Group): Promise<Group>;
     deleteGroup (gid: string): Promise<any>;
```

Comment:
I think we should keep this as `uids: string[]` and not user objects

Reply:
Yeah it wasn't working, so I had to change it.